### ISSUE TITLE
Reproduced results on miniImagenet with prototypical network with 4-layer network

### ISSUE THREAD

**Issue by kikyou123** [kjunelee/MetaOptNet#9]
Hi, many thanks for the sharing of the code.
I reproduced the results of the **prototypical network** (**4-layer network**)on miniImagenet, under python=2.7 with following packages.
torch==1.0.0.

I trained with only train dataset, and the best model is selected on the validation set.
I tested the best model on the test data, the results are:
Accuracy: (1-shot) 43.79 ± 0.65 --- your paper: ( 53.47 )

I am wondering if the difference is reasonable in this task? I am also wondering if the difference is due to my running environments.

Many thanks!!

**Comment by kjunelee** [comment]
I guess it is because you meta-trained with 15-shot or 10-shot episodes. As shown in Figure 2 of our paper, 1-shot performance of ProtoNet drops significantly if you meta-train with high shot.

**Comment by kikyou123** [comment]
Yes, I will try with 1-shot. Thanks very much!
